projects/plugins/jetpack/tools/webpack.config.search.js

```diff
@@ -75,4 +75,11 @@ module.exports = {
 			requestToHandle: defaultRequestToHandle,
 		} ),
 	],
+	optimization: {
+		splitChunks: {
+			cacheGroups: {
+				vendors: false,
```

Comment:
Note: I added this to avoid having a `vendors` chunk created for the lazily loaded main payload. It doesn't seem particularly useful, since there aren't a bunch of different dynamic chunks among which to share any vendor libs.